app/src/main/res/layout/details_fragment.xml

```diff
@@ -69,4 +69,17 @@
         android:layout_marginTop="492dp"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/GoBack"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:onClick="goBack"
+        android:text="Go back"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.775"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="OnClick" />
```

Comment:
This just part of the button I made to go back as I mention before.

Reply:
See previous comment.  Don't duplicate system functions.  

Or, consider using swipe gestures, so you're not adding another View to the screen.  More views == less simple.